src/mobile/components/layout/templates/head.jade

```diff
@@ -13,7 +13,12 @@ link( rel='apple-touch-icon', sizes='76x76', href=asset('/images/icon-76.png') )
 link( rel='apple-touch-icon', sizes='120x120', href=asset('/images/icon-120.png') )
 link( rel='apple-touch-icon', sizes='152x152', href=asset('/images/icon-152.png') )
 link( rel='apple-touch-icon', href=asset('/images/icon-152.png') )
-link( href="#{sd.APP_URL}#{sd.CURRENT_PATH}", rel="canonical" )
+
+if sd.PAGE_TYPE == "partner" && sd.PROFILE.id
```

Comment:
Technically this "does the thing" but it is pretty brittle and can scale out pretty quickly.


We are in a bit of a tight spot given the differences in the layout of these mobile views. In the mobile views, the canonical is set in the top level file, where in the desktop views, the canonical is set on a per app basis. See here for example (https://github.com/artsy/force/pull/6260/files). 

This is something we can further break down if thats something we are interested in but that would require some over head. Unless I am missing info, it looks like this is the sole file where any canonical info is handled for mobile views. 

Would love your thoughts, if this is something we could assess and mob on that could be fun too. There is another [moblie view bug](https://artsyproduct.atlassian.net/browse/PURCHASE-2120) coming down the pike that would introduce an additional conditional check in this file so we will be poking around here again soon.

Reply:
Considering mobile apps are legacy approach and we'll not adding more, it makes sense to me to keep the branching in a centralized place like here, with minimum changes.

Reply:
Thanks for the context!